dynamics/apis/api_base.py

```diff
@@ -234,3 +243,51 @@ def _delete_request(self, params, api_url):
 
         else:
             raise DynamicsError('Error: {0}'.format(response.status_code), response.text)
+
+    def _bulk_post_request(self, data):
+        """Create a HTTP batch request.
+
+        Parameters:
+            data (dict): HTTP POST body data for the wanted API.
+            api_url (str): Url for the wanted API.
```

Comment:
not able to see this parameter is this required ??

Reply:
No removed it